src/scenes/ModulesRouter/ModuleView.tsx

```diff
@@ -0,0 +1,169 @@
+      },
+    ],
+    [activeLocale, projectId]
+  );
+
+  useEffect(() => {
+    void dispatch(requestGetModule(moduleId));
+  }, [dispatch, moduleId]);
+
+  return (
+    <PageWithModuleTimeline crumbs={crumbs} hierarchicalCrumbs={false} title={<ModuleViewTitle module={module} />}>
+      <Card
+        sx={{
+          borderRadius: '24px',
+          boxShadow: 'none',
+          display: 'flex',
+          flexDirection: 'column',
+          flexGrow: 1,
+          marginBottom: theme.spacing(3),
+          padding: `${theme.spacing(2)} ${theme.spacing(1)}`,
+        }}
+      >
+        {module && (
+          <Grid container spacing={theme.spacing(1)}>
+            <Grid item xs style={{ flexGrow: 1, padding: `${theme.spacing(1)} ${theme.spacing(3)}` }}>
+              <ModuleContentSection>
+                <Typography fontSize={'16px'} lineHeight={'24px'} fontWeight={500}>
+                  {strings.formatString(strings.MODULE_NAME_OVERVIEW, module.name)}
+                </Typography>
+                <Typography fontSize={'24px'} lineHeight={'32px'} fontWeight={600}>
+                  {module.title}
+                </Typography>
+              </ModuleContentSection>
+
+              <ModuleContentSection>
+                <Box dangerouslySetInnerHTML={{ __html: module.description || '' }} />
+              </ModuleContentSection>
+
+              {module.contents.length && (
+                <>
+                  <ModuleContentSection>
+                    <Typography fontSize={'20px'} lineHeight={'28px'} fontWeight={600}>
+                      {strings.THIS_MODULE_CONTAINS}
+                    </Typography>
+                  </ModuleContentSection>
+
+                  {module.contents.map((content) => (
+                    <ModuleContentSection key={content.id}>
+                      <Link fontSize='16px' to='#'>
+                        {content.title}
+                      </Link>
+                      {content.dueDate && (
+                        <Typography fontSize={'16px'} lineHeight={'24px'} fontWeight={400}>
+                          {strings.formatString(
+                            strings.DUE,
+                            getLongDate(content.dueDate.replace('PST', ''), activeLocale)
+                          )}
+                        </Typography>
+                      )}
+                    </ModuleContentSection>
+                  ))}
+                </>
+              )}
+            </Grid>
+
+            {module.events.length && (
+              <Grid item>
+                {module.events.map((event) => (
+                  <ModuleFieldDisplay
+                    key={event.id}
+                    label={event.name}
+                    onClickButton={() => goToModuleEvent(projectId, event.id, module.id)}
+                    value={event.eventTime ? getLongDateTime(event.eventTime.replace('PST', ''), activeLocale) : ''}
```

Comment:
curious why the `replace`

Reply:
Oh great question, this was a hack because the datetime format in the mock data needs to be adjusted because it doesn't appear to be valid as-is. (JS date is unable to parse it and passing to one of the datetime formatter functions was throwing an error.) I'll fix the datetime format in the mock data is it's parseable by JS date and we can adjust the format if it isn't quite right.